advance-JavaScript/stack&queue.js

```diff
@@ -1,24 +1,16 @@
+// A stack is Last-In First-Out (LIFO) system.
 const Stack =  function(stacker){
   this.stacker = [];
 }
 Stack.prototype.add = function (arg){
-  this.stacker.unshift(arg);
+  this.stacker.push(arg);
```

Comment:
Changed this and the remove function to make the code a bit easier to read. For both Stack and Queue, you can use push to add elements and just change how you remove elements (`pop` vs `shift`).

Note: using `unshift` wasn't wrong. This is just to help point out the differences.